docs/TELEMETRY.md

```diff
@@ -31,133 +35,223 @@ This information is used to evaluate the quality of the model.
 
-This ping is sent every time an enrolled user performs a history / bookmark search in the awesome bar.
+Three main interactions with the awesome bar trigger a model update via study telemetry:
+
+1. A suggestion was selected from the autocomplete popup
+2. The autocomplete popup got some suggestions displayed but none were selected
+3. The autocomplete popup did not get some suggestions displayed and none was selected
+
 The following data is sent with this ping:
 
-| name                        | type              | description                                                                                         |
-|-----------------------------|-------------------|-----------------------------------------------------------------------------------------------------|
-| `model_version`             | integer           | the version of the model that all the other data is based on                                        |
-| `study_variation`           | string            | in what variation is the user enrolled in (e.g. treatment, control)                                 |
-| `update`                    | array of floats   | the model improvement that the user is proposing                                                    |
-| `loss`                      | float             | a number quantifying how well the model worked                                                      |
-| `num_suggestions_displayed` | integer           | how many history / bookmark suggestions were displayed?                                             |
-| `rank_selected`             | integer           | what was the position of the selected suggestion? (should be minimized by the optimization process) |
-| `num_chars_typed`           | integer           | how many characters did the user type? (should be minimized by the optimization process)            |
-| `frecency_scores`           | array of integers | what scores did the model assign to the suggestions?                                                |
+| name                                             | type              | description                                                                                                                                |
+| ------------------------------------------------ | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `model_version`                                  | integer           | the version of the model that all the other data is based on                                                                               |
+| `frecency_scores`                                | array of integers | what scores did the model assign to the suggestions?                                                                                       |
```

Comment:
@motin - this array should be floats as per https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/277/files#diff-5c2b8d38789e1c1028cafdb59c201e47R9

Reply:
Good catch!